backend/apps/organizations/signals.py

```diff
@@ -44,19 +44,20 @@ def handle_removed_member(instance: Membership, **kwargs):
 @receiver(post_save, sender=Organization)
 def create_default_groups(instance: Organization, created, **kwargs):
     """
-    Creates and assigns a primary group and HR group to members of the organization.
+    Creates and assigns a primary group and ADMIN group to members of the organization.
     """
     if created:
         ResponsibleGroup.objects.create(
-            name=PRIMARY_GROUP_NAME,
+            name=MEMBER_GROUP_NAME,
             description=f"Medlemmer av {instance.name}.",
             organization=instance,
-            group_type=PRIMARY_TYPE,
+            group_type=MEMBER_GROUP_TYPE,
         )
-        hr_group = ResponsibleGroup.objects.create(
-            name=HR_GROUP_NAME,
-            description=f"HR-gruppen til {instance.name}. Tillatelser for å se og behandle søknader.",
+        admin_group = ResponsibleGroup.objects.create(
+            name=ADMIN_GROUP_NAME,
+            description=f"ADMIN-gruppen til {instance.name}. Tillatelser for å se og behandle søknader og medlemmer.",
             organization=instance,
-            group_type=HR_TYPE,
+            group_type=ADMIN_GROUP_TYPE,
         )
-        assign_perm("forms.add_form", hr_group.group)
+        assign_perm("forms.add_form", admin_group.group)
+        assign_perm("organizations.manage_organization", admin_group.group)
```

Comment:
This is not really a big deal, but since you're limiting this permission to admins, just a heads up. 

If you have a user with two memberships, such as:

Rubberdøk: admin
Janus: member

The user would get the permissions from the admin group in Rubberdøk, including `organizations.manage_organization`. As a result, the user would be _also_ have permission to make changes to Janus, as the permission itself is not linked to the specific organization.

We do actually support per-object permissions, e.g. by doing (if I recall correctly), `assign_perm("organizations.manage_organization", admin_group.group, organization)`. However, in an effort to limit the scope of this PR, I suggest that you remove this line for now, and make a separate PR just for this. By doing that, there's less stuff that can break in an already complex PR 😊